packages/appstore/src/components/containers/__tests__/listing-container.spec.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ListingContainer from '../listing-container';
+import { StoreProvider, mockStore } from '@deriv/stores';
+
+jest.mock('Components/pre-loader/title-card-loader', () => jest.fn(() => <span>TitleCardLoader</span>));
+jest.mock('Components/currency-switcher-card', () => jest.fn(() => <span>CurrencySwitcherCard</span>));
+jest.mock('Components/containers/grid-container', () => jest.fn(({ children }) => <>{children}</>));
+
+const Title = () => <span>Test Title</span>;
+const Description = () => <span>Test Description</span>;
+
+const mock = mockStore({
+    client: {
+        is_landing_company_loaded: true,
+    },
+});
+
+describe('ListingContainer', () => {
+    const wrapper = ({ children }: { children: JSX.Element }) => <StoreProvider store={mock}>{children}</StoreProvider>;
+
+    it('should render title and description when landing company is loaded', () => {
+        mock.client.is_landing_company_loaded = true;
```

Comment:
We dont need this since its already true at the top 😸

Reply:
@thisyahlen-deriv changed line #13 because I'm sure would be better to set `is_landing_company_loaded` for the each test case separately if it influences on the result